src/test/java/org/openmrs/contrib/qaframework/page/AttachmentsPage.java

```diff
@@ -48,7 +53,7 @@ public String createRandomFile() throws IOException {
 		return tempFile.getAbsolutePath();
 	}
 	
-	public void setFileUrl()throws IOException {
+	public void setRandomFileUrl()throws IOException {
```

Comment:
@Irenyak1 step back and look at the method name critically.

Reply:
@kdaud would getRandomFileUrl() pass for this. I looked at it but thought between setRandomFileUrl() and getRandomFileUrl(), would work. What are your thoughts?

Reply:
For the created random file, is the method supposed be `getRandomFileUlr() `or `getRandomFile()`?

Reply:
@kdaud if I am to think of it now, it actually get the random file created since createRandomFile() returns the url for the created random file. So my method should be getRandomFile() to be precise.